packages/babel-plugin-minify-simplify/src/index.js

```diff
@@ -28,6 +28,62 @@ module.exports = ({ types: t }) => {
   const or = (a, b) => t.logicalExpression("||", a, b);
   const and = (a, b) => t.logicalExpression("&&", a, b);
 
+  const operators = new Set([
+    '+', '-', '*', '%',
+    '<<', '>>', '>>>',
+    '&', '|', '^', '/',
+    '**'
+  ]);
+
+  const updateOperators = new Set([
+    '+', '-'
+  ]);
+
+  function isEqual(arr1, arr2) {
+    return arr1.every((value, index) => {
+      return value.toString() === arr2[index].toString();
+    });
+  }
+
+  function getName(node) {
+    if (node.type === 'ThisExpression') {
```

Comment:
It's also about being able to use type aliases. So `t.isSomething` is a better check that you don't have to worry about whether it's an alias or not. If we forget that it's an alias and add some code, it'll be buggy and hard to debug later.